config/channels/in-memory-channel/webhooks/placeholder.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package webhooks is a placeholder that allows us to pull in config files
+// via go mod vendor.
+package webhooks
```

Comment:
Who do we need this file here?

Reply:
related to the vendoring, see old PR for this: https://github.com/knative/eventing/pull/3098/files

and latr it was renamed: https://github.com/knative/eventing/pull/4356/files